data_fusion/plugins/modules/procflows/data_fusion.py

```diff
@@ -621,15 +621,19 @@ def call(fnames, command_line_args=None):
         )
 
     retval = 0
-    from geoips.compare_outputs import compare_outputs
 
     if compare_path:
-        retval = compare_outputs(
-            compare_path.replace("<product>", final_product_name).replace(
-                "<procflow>", "data_fusion"
-            ),
-            final_products,
-        )
+        from geoips.interfaces.module_based.output_checkers import output_checkers
+
+        for output_product in final_products:
+            output_checker = output_checkers.get_plugin(output_product)
+            retval = output_checker(
+                output_checker,
+                compare_path.replace("<product>", final_product_name).replace(
+                    "<procflow>", "data_fusion"
+                ),
+                [output_product],
+            )
```

Comment:
Should this be a sum of all the retvals ?  Otherwise if the last product was 0, and an earlier product was non-zero, it would return 0?

Reply:
You're right. I modified it to only compare a single product at a time, which leaves the need to sum the return values. I'll change that.

Reply:
Great!